### PR TITLE
feat(PX-3911): return country full name on Contact tab

### PR DESCRIPTION
**Pull request by kiryl-zubarau**
This PR adds a dictionary of countries to show the full country name on the contact tab.

JIRA - https://artsyproduct.atlassian.net/browse/PX-3911

image

### CODE REVIEW

File: src/v2/Apps/Partner/Components/PartnerContacts/partnerContactUtils.ts [hunk 1]
```diff
@@ -66,3 +66,265 @@ export function getGoogleStaticMapImageUrl(
...
+  VU: "Vanuatu",
+  WF: "Wallis and Futuna",
+  WS: "Samoa",
+  YE: "Yemen",
+  YT: "Mayotte",
+  ZA: "South Africa",
+  ZM: "Zambia",
+  ZW: "Zimbabwe",
+  ZZ: "Unknown Region",
+}
```
Comment (dzucconi) [inline src/v2/Apps/Partner/Components/PartnerContacts/partnerContactUtils.ts:330]: This is definitely something that would be better suited to being in Metaphysics

Comment (kiryl-zubarau) [reply src/v2/Apps/Partner/Components/PartnerContacts/partnerContactUtils.ts:330]: Added branch: https://github.com/artsy/metaphysics/pull/3100

Will be updated after the PR merge.

Comment (kiryl-zubarau) [reply src/v2/Apps/Partner/Components/PartnerContacts/partnerContactUtils.ts]: Done!

Comment (dzucconi) [reply src/v2/Apps/Partner/Components/PartnerContacts/partnerContactUtils.ts]: 😗 👌

**Pull request merged**